oot.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/bash
+
+while getopts ':c:' OPT; do
```

Comment:
What's the first colon doing in the `optstring`? If I remove it `getopts` prints helpful error messages, which we might use to omit `Incorrect option` below altogether:

```console
$ ./oot/oot.sh -f
./oot/oot.sh: illegal option -- f
$ ./oot/oot.sh -c
./oot/oot.sh: option requires an argument -- c
```

Reply:
You might want to add `-f` too and set `FASTBOOT_FLASH=true` to replace the for loop in `compile.sh`.

Perhaps normalize the variable name to be consistent with the rest of the code, too.

Reply:
Done.

Reply:
@pablomh Looks like you only covered the second comment about `-f`; what's your preference/recommendation on the leading colon (`:`) in the optstring to print a helpful error message instead of doing it manually in `?)`?

Reply:
I re-read (again) what `:` implied and I agree with you that we don't need to set a different message for missing or wrong operands or options so I've removed it and left the default behaviour.